### PR TITLE
Add third saliency map interpretation notebook

### PR DESCRIPTION
**Pull request by GalyaZalesskaya**
The third part of showing [Explainable AI toolkit](https://github.com/openvinotoolkit/openvino_xai) functionality.
First part: https://github.com/openvinotoolkit/openvino_notebooks/pull/2150
Second part: https://github.com/openvinotoolkit/openvino_notebooks/pull/2159

### CODE REVIEW

**Comment by negvet** [comment]
todo @negvet: review results

**Comment by GalyaZalesskaya** [comment]
<p>Thank you for noticing. It wasn't updated after we changed the implementation in XAI. Fixed the same issue in 2nd deep-dive notebook.</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2240/discussion/#comment-2258159644'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by GalyaZalesskaya** [comment]
<p>This is the logger.info output that for some reason can't be captured by usual "%%capture" magic function. The other way to clean output is to add <code>setLevel</code> for logger, but it will require additional <code>logger</code> import, making the overall decision not concise. Maybe we can leave it as it is.  </p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2240/discussion/#comment-2258159616'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by GalyaZalesskaya** [comment]
<p>Will plan it for OVXAI 1.1 :)</p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2240/discussion/#comment-2258159758'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by negvet — APPROVED** [review]
Thank you for the notebook!

**Comment by GalyaZalesskaya** [comment]
@eaidova Hi Ekaterina, you've been a great help during the preparation of notebooks for XAI. Can you please review?

File: notebooks/explainable-ai-2-deep-dive/explainable-ai-2-deep-dive.ipynb [hunk 1]
```diff
@@ -4,16 +4,22 @@
    "cell_type": "markdown",
```
Comment (eaidova) [inline notebooks/explainable-ai-2-deep-dive/explainable-ai-2-deep-dive.ipynb:4]: <p>could you please remove this cell output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2240/discussion/#comment-1699427238'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='4'/>

File: notebooks/explainable-ai-3-map-interpretation/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Prepare IR model for inference
+  - Define preprocess and postprocess functions
+- Explain model using ImageNet labels
+- Explain for multiple images
+- Saliency map examples in different use cases and their interpretations
+  - True Positive High confidence
+  - True Positive Low confidence
+  - False Positive High confidence
+  - Two mixed predictions
```
Comment (eaidova) [inline notebooks/explainable-ai-3-map-interpretation/README.md:36]: please add installation section https://github.com/openvinotoolkit/openvino_notebooks/tree/latest/notebooks/action-recognition-webcam#installation-instructions and into notebook too

File: notebooks/explainable-ai-3-map-interpretation/imagewoof_use_cases.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
+{
+    "True_positive_high_confidence": {
```
Comment (eaidova) [inline notebooks/explainable-ai-3-map-interpretation/imagewoof_use_cases.json:2]: is it possible do not store this data in repository? In some cases notebooks users may do not clone repository and download only one notebook for running it instead (or open it with colab). Having extra files requires to be download in this case

**Comment by GalyaZalesskaya** [comment]
@eaidova Thank you for your great and thoughtful comments, I applied them

**Review by eaidova — APPROVED** [review]

**Pull request merged**